src/LoneSum.java

```diff
@@ -0,0 +1,23 @@
+public class LoneSum {
+    public static void main(String[] args) {
+        /** Given 3 int values, a b c, return their sum. However, if one of the values is the same as another of the values,
+         it does not count towards the sum.
+         */
```

Comment:
if you want this to be a 'dangling JavaDoc comment"\', it will be needed to be placed above a method definition, not a method call.